cmd/main.go

```diff
@@ -85,7 +85,9 @@ func Main() {
 	if err != nil {
 		fmt.Fprintf(os.Stderr, "%s: %v\n", subcommand.Name, err)
 	}
-
+	if err := cleanup(); err != nil {
```

Comment:
This should be called before any `os.Exit` otherwise it could still mess up things.

Reply:
I've moved it to the top of the function to just start with clean slate